python/taichi/math/mathimpl.py

```diff
@@ -0,0 +1,401 @@
+"""
+Math functions for glsl-like functions and other stuff.
+"""
+import taichi as ti
+
+pi = 3.1415926535897932
+e = 2.718281828459045
```

Comment:
may be better to use the constants from `math` module.

Reply:
Sure, will update this later.